catchup/service_test.go

```diff
@@ -421,6 +421,8 @@ func TestServiceFetchBlocksMalformed(t *testing.T) {
 }
 
 func TestOnSwitchToUnSupportedProtocol(t *testing.T) {
+	t.Skip("This test is flacky and need to be fixed.")
```

Comment:
There are a bunch of these, I think there's an umbrella ticket to fix:
```
=== Skipped
=== SKIP: agreement TestCryptoVerifierBuffers (0.00s)
    cryptoVerifier_test.go:139: Test is flaky

=== SKIP: agreement/gossip TestNetworkImplFullStackQuick (0.00s)
    networkFull_test.go:136: 

=== SKIP: config TestConfigMigrate (0.00s)
    config_test.go:247: 

=== SKIP: daemon/algod TestFirstListenerSetupGetsPort8080WhenPassedPortZero (0.00s)
    server_test.go:43: Cannot run this test since port 8080 is already in use.

=== SKIP: gen TestLoadMultiRootKeyConcurrent (0.00s)
    generate_test.go:37: 

=== SKIP: gen TestLoadSingleRootKeyConcurrent (0.00s)
    generate_test.go:78: 

=== SKIP: ledger TestArchival (0.00s)
    archival_test.go:127: Skipping the TestArchival as it tend to randomally fail on travis linux-amd64

=== SKIP: ledger TestLedgerMemoryLeak (0.00s)
    ledger_test.go:1587: 

=== SKIP: ledger TestRandomizedEncodingcatchpointFileBalancesChunk (0.00s)
    codec_tester.go:329: 

=== SKIP: ledger TestRandomizedEncodingencodedBalanceRecord (0.00s)
    codec_tester.go:329: 

=== SKIP: logging TestAsyncTelemetryHook_Close (0.00s)
    telemetryhook_test.go:174: We no longer ensure 100% delivery. To not block, we drop messages when they come in faster than the network sends them.

=== SKIP: logging TestAsyncTelemetryHook_QueueDepth (0.00s)
    telemetryhook_test.go:199: flakey test can fail on slow test systems

=== SKIP: node TestSyncingFullNode (0.00s)
    node_test.go:180: This is failing randomly again - PLEASE FIX!

=== SKIP: node TestInitialSync (0.00s)
    node_test.go:237: flaky TestInitialSync 

=== SKIP: node TestSimpleUpgrade (0.00s)
    node_test.go:270: Randomly failing: node_test.go:~330 : no block notification for account. Re-enable after agreement bug-fix pass

=== SKIP: rpcs TestStartAndStop (0.00s)
    txSyncer_test.go:263: TODO: replace this test in new client paradigm

=== SKIP: tools/network TestRealNamesWithResolver (0.00s)
    resolveController_test.go:76: 

=== SKIP: tools/network/dnssec TestDeadNS (0.00s)
    dnssec_test.go:315: 

=== SKIP: tools/network/dnssec TestRealRequests (0.00s)
    dnssec_test.go:332: 
```

Reply:
yes, I'm aware of that.. The important thing is that we need to keep tracking these explicitly so that we won't forget these.